### PR TITLE
[EZY KARMA]: remove outbound proxy skiplist FF

### PR DESCRIPTION
**Pull request by bgvozdev**
**What's in this PR?**
Removing FF

**Why**
Not used anymore: for the internal testing instance we should use API gateway

### CODE REVIEW

File: src/util/get-github-client-config.ts [hunk 1]
```diff
@@ -123,14 +90,14 @@ const buildGitHubClientServerConfig = async (gitHubServerApp: GitHubServerApp, j
 	}
 );
 
-const buildGitHubClientCloudConfig = async (jiraHost: string, logger: Logger): Promise<GitHubClientConfig> => {
+const buildGitHubClientCloudConfig = async (jiraHost: string): Promise<GitHubClientConfig> => {
```
Comment (krazziekay) [inline src/util/get-github-client-config.ts:93]: This won't be async too.

File: src/util/get-github-client-config.ts [hunk 2]
```diff
@@ -88,18 +55,18 @@ const buildGitHubServerConfig = async (githubServerBaseUrl: string, jiraHost: st
...
 		apiUrl: `${githubServerBaseUrl}/api/v3`,
 		graphqlUrl: `${githubServerBaseUrl}/api/graphql`,
-		proxyBaseUrl: await calculateProxyBaseUrl(jiraHost, githubServerBaseUrl, logger),
+		proxyBaseUrl: envVars.PROXY,
 		... await calculateApiKeyConfig(jiraHost, logger, apiKeyConfig)
 	};
 };
 
-const buildGitHubCloudConfig = async (jiraHost: string, logger: Logger): Promise<GitHubConfig> => {
+const buildGitHubCloudConfig = async (): Promise<GitHubConfig> => {
```
Comment (krazziekay) [inline src/util/get-github-client-config.ts:63]: This doesn't need to be async anymore.

**Review by krazziekay — COMMENTED** [review]
Minor changes

File: src/util/get-github-client-config.ts (hunk 1 above)
Comment (bgvozdev) [reply src/util/get-github-client-config.ts:93]: This one has to because of
`const privateKey = await keyLocator(undefined, jiraHost);`

**Review by krazziekay — APPROVED** [review]

**Pull request merged**